src/Promise.extensions.ts

```diff
@@ -0,0 +1,77 @@
+import 'core-js/features/promise';
```

Comment:
Can you add docs about browser compat/when to include this extensions file/etc.?  It appears that this has to be included from the caller bundle (which is great!). so just looking for guidance for users.

Reply:
Sure, I added some comments here, let me know if you need additional clarifications